data/ecosystem/integrations.yaml

```diff
@@ -227,3 +227,8 @@
   docsUrl: https://docs.konghq.com/mesh/latest/guides/otel-metrics/
   components: [Go]
   oss: false
+- name: Akka
+  url: https://akka.io/
+  docsUrl: https://developer.lightbend.com/docs/telemetry/current/plugins/opentelemetry/intro.html
```

Comment:
Reviewing this documentation it looks like that this only works with the Cinnamon Agent only?

Reply:
Correct. Cinnamon uses the OpenTelemetry client / api under the hood.

Reply:
From reading your documentation it sounds that (a) this is mandatory and (b) the agent is commercial?

Reply:
Yes, that seems right. If that is an issue, please let me know and I will close this pr.

Reply:
It's an unusual case what we have here. We have commercial integrations listed already, but we never had a situation where a commercial agent provides first party instrumentation for a library.